llvm/lib/SandboxIR/Type.cpp

```diff
@@ -47,7 +47,7 @@ void Type::dump() {
 
 PointerType *PointerType::get(Type *ElementType, unsigned AddressSpace) {
   return cast<PointerType>(ElementType->getContext().getType(
-      llvm::PointerType::get(ElementType->LLVMTy, AddressSpace)));
+      llvm::PointerType::get(ElementType->LLVMTy->getContext(), AddressSpace)));
 }
```

Comment:
We should remove these SandboxIR methods. They shouldn't have been copied into brand-new code...

Reply:
I can look into this later. From what I can tell there's only a few uses, all of which are related to SandboxIR

I'm not very familiar with SandboxIR, but from what I can tell it's relatively new. Do you think we also want a deprecation/removal process, or do you think it's fine to remove them?

Reply:
It's okay to remove them right away, no need to deprecate.